packages/notifi-react-card/lib/components/intercom/ChatWindowHeader.tsx

```diff
@@ -24,15 +24,15 @@ export const ChatWindowHeader: React.FC<ChatWindowHeaderProps> = ({
   return (
     <div
       className={clsx(
-        'NotifiIntercomChatWindowHeader__container',
+        'NotifiIntercomHeader__container',
```

Comment:
I believe that you had the correct styling when you first created that PR and were using the old CSS classname. Can you please let me know what caused the styling to change?

Reply:
yeah, I think I has changed the className in css file after the initial PR in order to use in another component, and forgot the change the className in ChatWindowHeader, should be a copy and paste mistake.

Reply:
Got it, thank you for the clarification!